src/main/java/edu/harvard/iq/dataverse/AbstractGlobalIdServiceBean.java

```diff
@@ -77,7 +77,20 @@ public String getIdentifierForLookup(String protocol, String authority, String i
         metadata.put("datacite.publisher", producerString);
         metadata.put("datacite.publicationyear", generateYear(dvObjectIn));
         return metadata;
-    }  
+    } 
+    
+    protected HashMap<String, String> addMetadataForDestroyedDataset(DvObject dvObjectIn) {
```

Comment:
Should the return type be `Map` instead of the more specific `HashMap`?

Reply:
Sure. Will change.